app/actions/views/file_upload.js

```diff
@@ -24,9 +24,9 @@ export function handleUploadFiles(files, rootId) {
             const uri = file.uri;
             const clientId = generateId();
 
-            if (extension === 'HEIC') {
+            if (extension.toUpperCase() === 'HEIC') {
                 extension = 'JPG';
-                name = name.replace(/HEIC/, 'jpg');
+                name = name.replace(/(HEIC|heic)/, 'jpg');
                 mimeType = 'image/jpeg';
             }
```

Comment:
You could also use the same case-insensitive regex for both the test and replace, but it's purely a personal preference:
```javascript
const re = /heic/i;
if (re.test(extension)) {
    extension = 'JPG';
    name = name.replace(re, 'jpg');
    mimeType = 'image/jpeg';
}
```

Reply:
Yeah maybe, I’ll merge this one for now and if you feel very strong about it you can create a ticket ;)

Thanks martin

Reply:
@enahum No totally not worth changing and I shouldn't have bothered mentioning if. You're all good 👍